backend/server/src/test/java/ai/verta/modeldb/IntegrationTest.java

```diff
@@ -180,22 +195,22 @@ private CreateExperimentRun createExperimentRunRequest(String projectId, String
             .build();
     metrics.add(
         KeyValue.newBuilder()
-            .setKey("metrics_" + Calendar.getInstance().getTimeInMillis())
+            .setKey("metrics_2_" + Calendar.getInstance().getTimeInMillis())
             .setValue(stringValue)
             .setValueType(ValueType.STRING)
             .build());
 
     List<Observation> observations = new ArrayList<>();
-    observations.add(
-        Observation.newBuilder()
-            .setArtifact(
-                Artifact.newBuilder()
-                    .setKey("Google developer Observation artifact")
-                    .setPath("This is data artifact type in Google developer Observation artifact")
-                    .setArtifactType(ArtifactType.DATA)
-                    .build())
-            .setTimestamp(Calendar.getInstance().getTimeInMillis())
-            .build());
+    /*observations.add(
+    Observation.newBuilder()
+        .setArtifact(
+            Artifact.newBuilder()
+                .setKey("Google developer Observation artifact")
+                .setPath("This is data artifact type in Google developer Observation artifact")
+                .setArtifactType(ArtifactType.DATA)
+                .build())
+        .setTimestamp(Calendar.getInstance().getTimeInMillis())
+        .build());*/
```

Comment:
should we just delete this code rather than comment it out?

Reply:
Removed